moto/batch/models.py

```diff
@@ -483,6 +483,7 @@ def __init__(
         self.job_stopped_at = datetime.datetime(1970, 1, 1)
         self.job_stopped = False
         self.job_stopped_reason: Optional[str] = None
+        self.raised_exception: Optional[Exception] = None
```

Comment:
Same for all changes in this file

Reply:
will check, I knew there was something I did miss...